media/js/firefox/whatsnew/whatsnew-exp-78.js

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function (Mozilla) {
+    'use strict';
+
+    /* update dataLayer with experiment info */
+    var href = window.location.href;
+    var platform = window.site.platform;
+    var isMobile = /^(android|ios|fxos)$/.test(platform);
+
+    var initTrafficCop = function () {
+        if (href.indexOf('v=') !== -1) {
+            if (href.indexOf('v=1') !== -1) {
+                window.dataLayer.push({
+                    'data-ex-variant': 'whatsnew-78-control',
+                    'data-ex-name': 'whatsnew-78-Experiment'
+                });
+            } else if (href.indexOf('v=2') !== -1) {
+                window.dataLayer.push({
```

Comment:
This script needs to be included in all the variation templates as well. currently it's only added to index.html, which means none of these dataLayer events get fired.